src/config.ts

```diff
@@ -4,6 +4,22 @@ import MemoryQueue from "./core/queue/memoryQueue";
 import { QueueInterface } from "./core/queue";
 import { StoreInterface } from "./core/store";
 import { HexString } from "./types/Strings";
+import { MissingProvider, MissingSigner, MissingStore, MissingUser } from "./core/utilities";
```

Comment:
Maybe we should just pull these errors into this file now that this is the only place they're thrown?